### DISCUSSION TITLE
Is it possible to create universal in-game driver for keyboard like for tablet?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#21400] [Q&A] [answered]
My strange idea: **Custom driver that uses only 3 keys (for std) during gameplay** (driver is disabled during pauses)

I don't know it can help with high latency on keyboard or not and I want to ask. Is it possible to create something like this? Can it help to reduce latency?
 _(I don't know if a developers would be interested in something like that. Therefore this is more of a question than a development proposal)._

**Comment by peppy** [comment, marked as answer]
If your keyboard has latency it's in the hardware. Buy a new keyboard.
